packages/next/client/index.js

```diff
@@ -99,15 +98,14 @@ export default async ({
     pageLoader,
     App,
     Component,
-    ErrorComponent,
     err: initialErr
   })
 
   router.subscribe(({ App, Component, props, err }) => {
     render({ App, Component, props, err, emitter })
   })
 
-  render({ App, Component, props, err: initialErr, emitter })
+  render({ App, Component, props, err: initialErr, emitter, router })
```

Comment:
Why is the router passed to render here 🤔

Reply:
Left over from a variant that used `fetchComponent`.

Reply:
👍